Shuttle-front/.editorconfig

```diff
@@ -3,8 +3,8 @@ root = true
 
 [*]
 charset = utf-8
-indent_style = space
-indent_size = 2
+indent_style = tab
```

Comment:
A tab could be a different number of columns depending on your IDE, but a space is always one column.

Reply:
I'll revert it to default, but in the case of VS code, the program will enforce this indentation regardless of what you put in, say, the JSON configuration for VS code. 
Fixed in [9479f1c](https://github.com/ili0n/Shuttle-front/pull/7/commits/9479f1ca2e8ccdaf1ffd2f32d9124a343ff8d5fd).